### PR TITLE
git commit -m "task created"

### PR DESCRIPTION
**Pull request by Nattalli**

### CODE REVIEW

File: app/test/conftest.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import pytest
+
+from django.core.management import call_command
+
+
+@pytest.fixture(scope='session')
```
Comment (Abnormaltype) [inline app/test/conftest.py:6]:
```suggestion
@pytest.fixture(scope="session")
```

File: app/test/conftest.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+def django_db_setup(django_db_setup, django_db_blocker):
+    with django_db_blocker.unblock():
+        call_command('loaddata', 'test_data.json')
```
Comment (Abnormaltype) [inline app/test/conftest.py:9]:
```suggestion
        call_command("loaddata", "test_data.json")
```

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Run flake8
         run: flake8 app/
 
-      - name: Run tests
+      - name: Run test
```
Comment (Abnormaltype) [inline .github/workflows/test.yml:30]: why did you do this?

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 3 above)
Comment (Nattalli) [reply .github/workflows/test.yml:30]: I thought that it will fix my problem, but indeed it is not:)

**Review by Abnormaltype — APPROVED** [review]

**Pull request merged**